go.md

```diff
@@ -0,0 +1,24 @@
+# Working with Go language versions in project
+
+Upon the release of a new version of Go, all repositories are updated
+according to the following conditions:
+* The minimum supported version of Go is set to two versions back from 
+the previous one. (For example, upon the release of Go 1.22, the minimum 
+version becomes 1.20.)
+* All dependency packages are updated (sometimes, according to project 
+requirements, some deprecated package methods may be updated as well.)
```

Comment:
If we write about dependencies and deprecated packages, then maybe it should also be mentioned that we need to adapt new Go version features?

Reply:
Renamed. Do you want me to add an index in the README?